httpclient5/src/test/java/org/apache/hc/client5/http/psl/TestPublicSuffixMatcher.java

```diff
@@ -46,36 +44,78 @@ public void setUp() throws Exception {
         final ClassLoader classLoader = getClass().getClassLoader();
         final InputStream in = classLoader.getResourceAsStream(SOURCE_FILE);
         Assert.assertNotNull(in);
-        final PublicSuffixList suffixList;
-        try {
-            final PublicSuffixListParser parser = new PublicSuffixListParser();
-            suffixList = parser.parse(new InputStreamReader(in, StandardCharsets.UTF_8));
-        } finally {
-            in.close();
-        }
-        matcher = new PublicSuffixMatcher(suffixList.getRules(), suffixList.getExceptions());
+        matcher = PublicSuffixMatcherLoader.load(in);
     }
 
     @Test
-    public void testGetDomainRoot() throws Exception {
+    public void testGetDomainRootAnyType() {
+        // Private
         Assert.assertEquals("example.xx", matcher.getDomainRoot("example.XX"));
         Assert.assertEquals("example.xx", matcher.getDomainRoot("www.example.XX"));
         Assert.assertEquals("example.xx", matcher.getDomainRoot("www.blah.blah.example.XX"));
-        Assert.assertEquals(null, matcher.getDomainRoot("xx"));
-        Assert.assertEquals(null, matcher.getDomainRoot("jp"));
-        Assert.assertEquals(null, matcher.getDomainRoot("ac.jp"));
-        Assert.assertEquals(null, matcher.getDomainRoot("any.tokyo.jp"));
+        // Too short
+        Assert.assertNull(matcher.getDomainRoot("xx"));
```

Comment:
@nielsbasjes I would prefer to keep non-functional changes in a separate commit.

Reply:
I reverted this change to assertNull